flink-streaming-connectors/flink-connector-kafka/src/main/java/org/apache/flink/streaming/connectors/kafka/FlinkKafkaConsumer.java

```diff
@@ -449,6 +481,8 @@ public void close() throws Exception {
 
 	@Override
 	public long[] snapshotState(long checkpointId, long checkpointTimestamp) throws Exception {
+		Preconditions.checkState(offsetCommitter == null, "Periodic offset committer and checkpointing can not be enabled at the same time");
```

Comment:
The other preconditions functions are statically imported. Would be nice to follow one style within a class.